src/main/java/org/hl7/davinci/priorauth/ClaimEndpoint.java

```diff
@@ -202,7 +234,8 @@ private IBaseResource processBundle(Bundle bundle) {
     // TODO response.setPreAuthPeriod(period)?
```

Comment:
Should the disposition be something other than "Granted" maybe "Cancelled" as well?

Reply:
Yea I think we should. From the profile in the IG it just says :

> A human readable description of the status of the adjudication.

After a cancel I will update to have the disposition say "Cancelled"

Reply:
Updated: https://github.com/HL7-DaVinci/prior-auth/commit/4d3ae1bda03adbaba03855e7e8419e8d90ca8785